GoogleForCreators/web-stories-wp
# Media: Fix broken video muting

## Context



It seems like video muting got broken because the new file name (of the muted version) we sent to WP did not include the extension, so WP blocks it.

We haven't changed that code in a long time, so maybe this got broken because of a change on the WP side.

## Summary



This change ensures that the newly muted video file name includes the file extension.

## Relevant Technical Choices



## To-do



## User-facing changes



No visual changes

## Testing Instructions






This PR can be tested by following these steps:

1. Insert a video from the library or record one
2. Try to remove audio via the style panel
3. Verify that removing audio works.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #12040
